sqlglot/parser.py

```diff
@@ -3195,6 +3199,12 @@ def _parse_into(self) -> t.Optional[exp.Into]:
             exp.Into, this=self._parse_table(schema=True), temporary=temp, unlogged=unlogged
         )
 
+    def _parse_bulk_collect_into(self) -> t.Optional[exp.BulkCollectInto]:
+        if not self._match(TokenType.BULK_COLLECT_INTO):
+            return None
+
+        return self.expression(exp.BulkCollectInto, this=self._parse_table(schema=True))
```

Comment:
this doesnt' seem correct, it looks it can be a list of stuff right?

Reply:
@tobymao I'm pretty sure it is always into a single variable, not a list. I will confirm tomorrow.